metric/doc.go

```diff
@@ -26,9 +26,47 @@ instruments are created by a [Meter] which itself is created by a
 as a starting point when instrumenting. This can be done directly, or by using
 the OpenTelemetry global MeterProvider via [GetMeterProvider]. Using an
 appropriately named [Meter] from the accepted [MeterProvider], instrumentation
-can then be built from the [Meter]'s instruments. See
-[go.opentelemetry.io/otel/metric/instrument] for documentation on each
-instrument and its intended use.
+can then be built from the [Meter]'s instruments.
+
+# Instruments
+
+Each instrument is designed to make measurements of a particular type. Broadly,
+all instruments fall into two overlapping logical categories: asynchronous or
+synchronous, and int64 or float64.
+
+All synchronous instruments ([Int64Counter], [Int64UpDownCounter],
```

Comment:
Should we tell that there is no synchronous gauge and one can use `UpDownCounter` instead?
Example metric: "room temperature goal" (in an air conditioner setting, for comparison a "actual room temperature" can be an async gauge.

Reply:
Same here:

> This is a copy/paste from the other package. I'd rather not edit this in this PR.
>
> Feel free to add a PR once this merges.